src/prefect_server/services/loop_service.py

```diff
@@ -19,6 +19,9 @@ class LoopService:
     # if no loop_seconds_config_key is provided, this will be the default
     loop_seconds_default = 600
 
+    # shutdown flag for gracefully exiting the infinite loop
+    is_running = True
```

Comment:
Does it make sense to have this be at the class level?

e.g.

```
In [1]: class foo:
   ...:     x = True
   ...: 

In [2]: foo.x
Out[2]: True

In [3]: instance = foo()

In [4]: instance.x
Out[4]: True

In [5]: foo.x = False

In [6]: instance.x
Out[6]: False
```

Reply:
Great question - all services which run in the same process will shutdown simultaneously either way (there's no need to selectively shut down a single service), so it's not necessary to make it an instance variable.

Reply:
Might be worth noting in a comment but that makes sense!